### ISSUE TITLE
Set keyboard focus on graph/table when selecting an analysis

### ISSUE THREAD

**Issue by hriday-panchasara** [eclipse-cdt-cloud/vscode-trace-extension#114]
PR 612 in `theia-trace-extension` (https://github.com/eclipse-cdt-cloud/theia-trace-extension/pull/612) provides the following features:
- Keyboard focus (indicated by a blue/orange border) is given to an output when its opened for the first time
- When clicking on an analysis title in the trace tab, keyboard focus should be the analysis' graph/chart region. 
- If a View that is already open is selected in the sidebar, the View's title area should pulse to temporarily bring attention to the View panel. Keyboard focus is also given to the analysis' graph/chart region.

Not all features provided by this patch work in vscode-trace-extension. Replicate the missing functionality
